frontend/src/pages/RegionalCommunicationLogDashboard/components/RegionalCommLogTable.js

```diff
@@ -0,0 +1,267 @@
+import useWidgetMenuItems from '../../../hooks/useWidgetMenuItems';
+import UserContext from '../../../UserContext';
+import useWidgetExport from '../../../hooks/useWidgetExport';
+import useWidgetSorting from '../../../hooks/useWidgetSorting';
+import { EMPTY_ARRAY } from '../../../Constants';
+import { deleteCommunicationLogById, getCommunicationLogs } from '../../../fetchers/communicationLog';
+import AppLoadingContext from '../../../AppLoadingContext';
+import { UsersIcon } from '../../../components/icons';
+import HorizontalTableWidget from '../../../widgets/HorizontalTableWidget';
+
+const COMMUNICATION_LOG_PER_PAGE = 10;
+
+const DEFAULT_SORT_CONFIG = {
+  sortBy: 'communicationDate',
+  direction: 'desc',
+  activePage: 1,
+};
+
+const headers = ['Recipient', 'Date', 'Purpose', 'Goals', 'Creator name', 'Other TTA staff', 'Result'];
+const headersForExporting = [...headers, 'Region', 'Recipient next steps', 'Specialist next steps', 'Files'];
+
+const DeleteLogModal = ({
+  modalRef,
+  onLogRemoved,
+  log,
+}) => {
+  const onDeleteLog = () => {
+    onLogRemoved(log)
+      .then(modalRef.current.toggleModal(false));
+  };
+
+  return (
+    <>
+      <Modal
+        modalRef={modalRef}
+        onOk={onDeleteLog}
+        modalId="DeleteLogModal"
+        title="Are you sure you want to delete this log?"
+        okButtonText="Delete"
+        okButtonAriaLabel="Confirm delete and reload page"
+        showCloseX
+      >
+        <p>This action cannot be undone.</p>
+      </Modal>
+    </>
+  );
+};
+
+DeleteLogModal.propTypes = {
+  modalRef: PropTypes.oneOfType([
+    PropTypes.func,
+    PropTypes.shape(),
+  ]).isRequired,
+  onLogRemoved: PropTypes.func.isRequired,
+  log: PropTypes.shape({
+    id: PropTypes.number,
+    displayId: PropTypes.string,
+    data: PropTypes.shape({
+      communicationDate: PropTypes.string,
+      purpose: PropTypes.string,
+      result: PropTypes.string,
+    }),
+    userId: PropTypes.number,
+  }),
+};
+
+DeleteLogModal.defaultProps = {
+  log: null,
+};
+
+export default function RegionalCommLogTable({ filters }) {
+  const [logs, setLogs] = useState();
+  const [error, setError] = useState();
+  const [showTabularData, setShowTabularData] = useState(true);
+  const [checkboxes, setCheckboxes] = useState({});
+  const [tabularData, setTabularData] = useState([]);
+  const [logToDelete, setLogToDelete] = useState(null);
+  const modalRef = useRef();
+  const history = useHistory();
+
+  const { user } = useContext(UserContext);
+  const { setIsAppLoading } = useContext(AppLoadingContext);
+
+  const { exportRows } = useWidgetExport(
+    tabularData,
+    headersForExporting,
+    checkboxes,
+    'Log ID',
+    'Communication_Log_Export',
+  );
+
+  const menuItems = useWidgetMenuItems(
+    showTabularData,
+    setShowTabularData,
+    null, // capture function
+    checkboxes,
+    exportRows,
+  ).filter((m) => !m.label.includes('Display'));
```

Comment:
this removes the display chart item, but we should probably refactor this hook if we plan to use this table anywhere else